### ISSUE TITLE
Pagination number losses its accessibility when is focused

### ISSUE THREAD

**Issue by allendol** [appbaseio/reactivesearch#1133]
When you focus on any buttons on pagination, it losses its accessibility. Cannot read the number any more. Combination of dark blue and black number is not readable.
![PaginationNumber](https://user-images.githubusercontent.com/40170767/64033475-8b68cc00-cb1a-11e9-834d-201d046bb9dd.png)

**Comment by jyash97** [comment]
Hey @allendol 
Check this example of [ReactiveList](https://codesandbox.io/s/github/appbaseio/reactivesearch/tree/next/packages/web/examples/ReactiveList?from-embed) I can see the color of the active page is white. Can you create a codesandbox link for the issue?

**Comment by allendol** [comment]
Hi @jyash97
For some reason I cannot see the code example you sent the link. Can you resend the link?

**Comment by allendol** [comment]
Hi @jyash97
I found other example and saw the focus is working fine. The issue was in my code.
Thanks!
